src/components/homePage/Testimonials.vue

```diff
@@ -1,19 +1,236 @@
 <template>
-<div id="testimonials">
-	<h2>Testimonials</h2>
-	<!-- SIDE SCROLLER -->
-</div>
+  <div id="testimonials">
+    <div>
+      <img src="../../assets/testimonial_building.svg" id="building" />
+      <vueper-slides 
+        class="no-shadow" 
+        :bullets="false" 
+        :arrows="false" 
```

Comment:
Since the slides aren't touchable (for now), it would be nice to add back the arrows. We can talk about whether this is required if you'd like, but my initial vote is to put them in!